SignalServiceKit/src/Contacts/Contact.m

```diff
@@ -65,6 +40,7 @@ - (instancetype)initWithSystemContact:(CNContact *)contact
     _cnContact = contact;
     _firstName = [self trimName:contact.givenName];
     _lastName = [self trimName:contact.familyName];
+    _fullName = [CNContactFormatter stringFromContact:contact style:CNContactFormatterStyleFullName];
```

Comment:
We need to test this.  Let's discuss what the complete set of test scenarios should be.